scripts/test_mlflow_prompts_manual.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+"""Manual test script for MLflow Prompts Provider.
+
+This script provides a quick way to manually test the MLflow prompts provider
+without running the full test suite.
+
+Prerequisites:
+    1. MLflow server running (default: http://localhost:5555)
+    2. MLflow package installed: pip install 'mlflow>=3.4.0'
+
+Usage:
+    # Start MLflow server (in separate terminal)
+    mlflow server --host 127.0.0.1 --port 5555
+
+    # Run this script
+    uv run python scripts/test_mlflow_prompts_manual.py
+
+    # Or with custom MLflow URI
+    MLFLOW_TRACKING_URI=http://localhost:8080 uv run python scripts/test_mlflow_prompts_manual.py
+"""
+
+import asyncio
+import os
+import sys
+
+
+async def main():
```

Comment:
i don't think we need this file, instead it should be in an integration test in the CI